src/platform/common/serviceRegistry.node.ts

```diff
@@ -73,7 +73,7 @@ export function registerTypes(serviceManager: IServiceManager) {
     serviceManager.addSingleton<ILanguageService>(ILanguageService, LanguageService);
     serviceManager.addSingleton<IBrowserService>(IBrowserService, BrowserService);
     serviceManager.addSingleton<IHttpClient>(IHttpClient, HttpClient);
-    serviceManager.addSingleton<ICryptoUtils>(ICryptoUtils, CryptoUtils);
+    serviceManager.addSingleton<ICryptoUtils>(ICryptoUtils, CryptoUtilsNode);
```

Comment:
Looking at the PR title, i don't see any code where we're dropping the use of the old fnv liv for hashing on the web.
This PR merely introduces a dedicated method for desktop and not the former.

Reply:
`CryptoUtils` is now only used in the Web and it doesn't use `fnv` anymore.